src/organisation/containers/organisation/organisation.component.html

```diff
@@ -29,6 +29,16 @@
       </td>
     </tr>
     <!-- !PBA Numbers -->
+    <tr class="govuk-table__row" *ngIf="!organisationPaymentAccount">
+      <th class="govuk-table__header" scope="row">PBA number</th>
+      <td class="govuk-table__cell">
+        <div><p>Until you’ve added one or more payment by account (PBA) numbers,
```

Comment:
@ishOsw - one small change, swap the `’` for a regular `'`... sometimes the "curly" version doesn't render properly on some platforms